src/components/tutorial.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dialog from 'modal-form/dialog';
+import MediaCard from './media-card'
+import { Markdown } from 'markdownz';
+import apiClient from 'panoptes-client/lib/api-client';
+import StepThrough from '../components/step-through';
+
+
+const completedThisSession = {};
+if (window.tutorialsCompletedThisSession) {
+  window.tutorialsCompletedThisSession = completedThisSession;
+}
+
+class Tutorial extends React.Component {
+  componentWillMount() {
+    this.previousActiveElement = document.activeElement;
+  }
+
+  componentWillUnmount() {
+    this.handleUnmount();
+  }
+
+  handleNextClick() {
+    this.stepThrough.goNext();
+  }
+
+  handleUnmount() {
+    if (this.previousActiveElement.focus) {
+      this.previousActiveElement.focus();
+    }
+    const now = new Date().toISOString();
+    completedThisSession[this.props.tutorial.id] = now;
+
+    if (this.props.user) {
+      const projectPreferences = this.props.preferences;
+      if (!projectPreferences.preferences) {
+        projectPreferences.preferences = {};
+      };
+      if (!projectPreferences.preferences.tutorials_completed_at) {
+        projectPreferences.preferences.tutorials_completed_at = {};
+      };
+      projectPreferences.update(`preferences.tutorials_completed_at.${this.props.tutorial.id}`: now);
```

Comment:
I think you might be missing `{}` around the object here, which would explain the error on save.

Reply:
This appears to be the root of the problem. It's a bit strange how this is constructed with CoffeeScript vs. ES^, but I'll push up a fix.